Documentation/contributing.rst

```diff
@@ -653,6 +723,60 @@ saved, in case of a failing test an exhaustive data will be added.
 	Allocated IPv4 addresses:
 
 
+Running with delve
+^^^^^^^^^^^^^^^^^^
+
+`Delve <https://github.com/derekparker/delve>`_ is a debugging tool for Go
+applications. If you want to run your test with delve,  you should add a new
+breakpoint using
+`runtime.BreakPoint() <https://golang.org/pkg/runtime/#Breakpoint>`_ in the
+code, and run ginkgo using ``dlv``.
+
+Example how to run ginkgo using ``dlv``:
+
+::
+
+	dlv test . -- --ginkgo.focus="Runtime" -ginkgo.v=true --cilium.provision=false
+
+
+Running test on other vms
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to run test in a different VM, you can use ``--cilium.SSHConfig``
+option to connect over ssh to another server. Cilium source code should be
+located on ``/home/vagrant/go/src/github.com/cilium/cilium/`` and test will run
+over the given SSH Connection. Also the target server should have Cilium
+installed and running.
+
+The ssh connection need to be defined as a ``ssh-config`` file and need to have
+the following targets:
+
+- runtime: To run runtime test.
+- k8s{1..2}-${K8S_VERSION}: to run Kubernetes test. It needs a Kubernetes installation
```

Comment:
Kubernetes tests. These instances must have Kubernetes installed and running as a prerequisite for running tests.